gene2phenotype_project/gene2phenotype_app/utils/curationinfo_utils.py

```diff
@@ -0,0 +1,78 @@
+from email.message import EmailMessage
+from smtplib import SMTP
+from django.conf import settings
+from django.template.loader import render_to_string
+from typing import Dict, Any
+
+class ConfidenceCustomMail():
+
+    @staticmethod
+    def send_confidence_update_email(instance: object, old_confidence: str, user_updated: str, request: object) -> None:
+        """
+            Confidence email update mail setup
+
+            Args:
+                instance (object): Instance object
+                old_confidence (str): old confidence object
+                user_updated (str): user that updated the confidence
+                request (object): the request object
+
+            Returns:
+                An exception, if there is an error in sending the mail or returns None
+        """
+
+        stable_id = instance.stable_id.stable_id
+        g2p_url = ConfidenceCustomMail.create_url_record(stable_id,request)
+        email_body = render_to_string('gene2phenotype_app/confidence_change_email.tpl', {
```

Comment:
This methods returns:
`django.template.exceptions.TemplateDoesNotExist: gene2phenotype_app/confidence_change_email.tpl`
The template is missing.